evalml/utils/woodwork_utils.py

```diff
@@ -82,9 +82,20 @@ def infer_feature_types(data, feature_types=None):
         return data
 
     if isinstance(data, pd.Series):
+        if all([isinstance(x, type(pd.NA)) for x in data]):
+            data = data.replace(pd.NA, np.nan)
+            feature_types = "Double"
         return ww.init_series(data, logical_type=feature_types)
     else:
         ww_data = data.copy()
+        # Revert the inference of all nulls to the unknown type and change it back to double.
+        all_null_cols = ww_data.columns[ww_data.isnull().all(0)]
```

Comment:
I don't think this will work if ww is initialized before being passed into one of our components?

![image](https://user-images.githubusercontent.com/41651716/129061176-efc50c56-dc0d-403a-8627-78218692a661.png)

I think this might also cause a problem with partial dependence but I have not verified. Down to talk about it after stand-up!

Reply:
I handled that specific case.  Feel free to check it out.  I was considering maybe adding a similar test for the target imputer with the y series being pre-inited.